mapfishapp/src/main/java/org/georchestra/mapfishapp/ws/DocController.java

```diff
@@ -358,7 +358,7 @@ private void storeFile(A_DocService docService, String docUrl, HttpServletReques
 
             // send back to client the url path to retrieve this file later on
             response.setStatus(HttpServletResponse.SC_CREATED); // 201 created, new resource created
-            response.setContentType("text/html; charset=utf-8");
+            response.setContentType("application/json");
```

Comment:
The change broke AJAX Form request.
From http://docs.sencha.com/extjs/3.4.0/#!/api/Ext.form.BasicForm : 

> The server response is parsed by the browser to create the document for the IFRAME. If the server is using JSON to send the return object, then the Content-Type header must be set to "text/html" in order to tell the browser to insert the text unchanged into the document body.